relay-server/src/actors/processor.rs

```diff
@@ -2907,20 +2930,25 @@ mod tests {
     }
 
     fn create_test_processor(config: Config) -> EnvelopeProcessorService {
+        let (envelope_manager, _) = mock_service("envelope_manager", (), |&mut (), _| {});
+        let (outcome_aggregator, _) = mock_service("outcome_aggregator", (), |&mut (), _| {});
+        let (project_cache, _) = mock_service("project_cache", (), |&mut (), _| {});
+        let (upstream_relay, _) = mock_service("upstream_relay", (), |&mut (), _| {});
```

Comment:
nit: Should we make the third argument to `mock_service` optional, or create a wrapper `noop_service` or similar that takes only one argument?

Reply:
yeah, I will look into this in the follow-ups